frontend/src/Components/RegistrationForm.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import FloatingLabel from 'react-bootstrap/FloatingLabel';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import '../CSS/registrationForm.css';
+
+const RegistrationForm = () => {
+  return (
+    <div className='registration-form-container'>
+        <h1 name='heading'>Create an account</h1>
+        <imgage>Upload a pic </imgage>
+        <form className='registration-form-input-container'>
+            <FloatingLabel
+                controlId="floatingInput"
+                label="Email Address"
+                className="mb-3"
+            >
+                  <Form.Control type="email" placeholder="name@example.com" name="email" required/>
+            </FloatingLabel>
+            <FloatingLabel
+                controlId="floatingInput"
+                label="First Name"
+                className="mb-3"
+            >
+                  <Form.Control type="text" placeholder="First Name" name='firstname' required/>
+            </FloatingLabel>
+            <FloatingLabel
+                controlId="floatingInput"
+                label="Last Name"
+                className="mb-3"
+            >
+                  <Form.Control type="text" placeholder="Last Name" name='lastname' required/>
+            </FloatingLabel>
+              <FloatingLabel
+                  controlId="Floating Password"
```

Comment:
Keep the naming convention for the attributes consistent - consider rewording this to floatingPassword.